src/checkin/services/flow/step/GetExtraPassengerInformationStep.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { MockContext } from '@testMocks/model/Context.mock'
+import { Context } from '../../../model/Context'
+import GetExtraPassengerInformationStep from './GetExtraPassengerInformationStep'
+
+const MOCK_NAMES = ['Carlos', 'Juan', 'Emilio', 'Gary', 'Elias', 'Fernando', 'Giovanni']
+const MOCK_PASSPORTS = ['G143327', 'G113125', 'G229108', 'G229101']
+
+describe('[ Step / GetExtraPassengerInformationStep ]', () => {
+  const step = new GetExtraPassengerInformationStep()
+
+  describe('Test when it should be execute', () => {
+    it('should return true when passengers are more than one', () => {
```

Comment:
What should return true? The step? I mean, does the step called "GetExtraPassengerInformation" have a boolean return value? I think we could improve the naming of test cases 😃

Reply:
Totally agree, this test can be more descriptive. Thanks!